KoreBuild-dotnet/build/_xunit-test.shade

```diff
@@ -51,6 +51,9 @@ projectFile=''
                         testArgs = " -parallel none";
                     }
 
+                    // Removing app domains to allow custom test discovery
+                    testArgs += " -noappdomain";
```

Comment:
Last I checked, this causes most 99 of 109 tests to fail in ["Microsoft.EntityFrameworkCore.Commands.FunctionalTests"](https://github.com/aspnet/EntityFramework/tree/dev/test/Microsoft.EntityFrameworkCore.Commands.FunctionalTests)